int64 id
int64 age
string test
string type
string accuracy